### DISCUSSION TITLE
Is monthPicker available

### DISCUSSION THREAD

**Discussion by 17euec129** [dmuy/duDatepicker#12] [General]
In my user experience, this dudatepicker is simple and excellence
So I would like to use this as monthpicker and yearpicker too, not only as datepicker
Suggest me if this feature is available or possibilty of getting this worked

Thanks In Advance
Sankar

**Comment by dmuy** [comment]
Hi @17euec129,

I'm sorry I didn't see any notifications on this thread. As for you question, this feature is currently **not available** but can be worked on. Maybe I'll include this in the future updates.

**Reply by dev-akshayyadav** [reply]
is it available. if (available == true)
{
  please show how to use it
}
